.gitignore

```diff
@@ -2,3 +2,4 @@
 *.log
 target/
 */target/
+.idea
```

Comment:
Ideally this should go in your global `.gitignore` (see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).
Using Intellij is the developer choice and I'd rather avoid listing all the work files of the many editors out there.

Reply:
Wow, you're right, didn't think about this